virtcontainers/persist/api/version.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2018 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package persistapi
+
+const (
+	// CurPersistVersion is current persist data version.
+	// This can help keep backward compatibility, if you make
+	// some changes in persistapi package which needs different
+	// handling process between different runtime versions, you
+	// should modify `CurPersistVersion` and handle persist data
+	// according to it.
+	// If you can't be sure if the change in persistapi package
+	// requires a bump of CurPersistVersion or not, do it for peace!
+	// --@WeiZhang555
+	CurPersistVersion uint = 1
```

Comment:
This value is added to the on-disk persistence state. However I think we need a basic check on the value when it is `Restore()`'d:

```go
if onDiskCurPersistVersion > CurPersistVersion {
    return fmt.Errorf("can only handle persistence data versions up to %d (got %d)", CurPersistVersion, onDiskCurPersistVersion)
}

Reply:
Yep, it's not finished. There should be a check for `CurPersistVersion`